src/content/tutorials/fleet-management/cluster-management/migration-to-cluster-api/_index.md

```diff
@@ -56,6 +56,69 @@ Our engineers will check that all resources and infrastructure are correctly mig
 - Some customers have been using [k8s-initiator-app](https://github.com/giantswarm/k8s-initiator-app/) to configure some aspects of the workload cluster API.
   In the new Cluster API implementation, [most of the features enabled by the app](https://github.com/giantswarm/capi-migration-cli/tree/main/k8s-initiator-features) are now supported natively by the platform. The app should be removed and moved to the new syntax if our migration CLI doesn't handle your use-case. Giant Swarm account engineers will help you with this process.
 
+- [Service Account issuer switch](#service-account-issuer-switch)
+
+- [Cluster manifest clean-up](#cluster-manifest-clean-up)
+
+### Service Account issuer switch
+
+For context, a cluster Service Account issuer is an OIDC provider that signs and issues the tokens for the cluster's Service Accounts. In turn, these Service Accounts can be used to authenticate workloads against the Kubernetes API and against the AWS API via IRSA.
+
+In CAPA, since the cluster domain name changes from Vintage, a new Service Account issuer is also introduced. To make for a smooth migration though, we support defining multiple issuers in a cluster. This way, Service Account tokens issued by all the defined issuers will be accepted in the cluster's Kubernetes API. The accepted issuers are defined in `cluster.providerIntegration.controlPlane.kubeadmConfig.clusterConfiguration.apiServer.serviceAccountIssuers` in the cluster App values (order matters as we'll see below).
+
+After a cluster is migrated to CAPI, the Vintage Service Account issuer needs to be phased out, since it's tied to the Vintage cluster domain name, which is also to be phased out eventually. This is a gradual, multi-step process that will require rolling the master nodes in multiple phases.
+
+1. Update the trust policy for all the AWS IAM Roles used by a Service Account via IRSA. The trust policy should allow `sts:AssumeRoleWithWebIdentity` for both Vintage and CAPA issuers during the transition period. This is how the trust policy of the IAM roles would look like (example, make sure to use the correct AWS account id, issuer domains and `ServiceAccount` references):
```

Comment:
```suggestion
After a cluster is migrated to CAPI, the vintage service account issuer needs to be phased out, since it's tied to the vintage cluster domain name, which will also be phased out eventually. This is a gradual, multi-step process that will require rolling the master nodes in multiple phases.

1. Update the trust policy for all the AWS IAM Roles used by a `ServiceAccount` via IRSA. The trust policy should allow `sts:AssumeRoleWithWebIdentity` for both Vintage and CAPA issuers during the transition period. This is how the trust policy of the IAM roles would look like (for example, make sure to use the correct AWS account id, issuer domains and `ServiceAccount` references):
```